packages/headless/src/features/commerce/search/search-analytics-actions.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  CommerceSearchAction,
+  makeCommerceAnalyticsAction,
+} from '../../analytics/analytics-utils';
+import {OmniboxSuggestionMetadata} from '../../query-suggest/query-suggest-analytics-actions';
+
+export const logInterfaceLoad = (): CommerceSearchAction =>
+  makeCommerceAnalyticsAction({
+    prefix: 'analytics/commerce/interface/load',
+    __legacy__getBuilder: (client) => client.makeInterfaceLoad(),
+    analyticsType: '',
+    analyticsPayloadBuilder: (_state) => {
+      return {};
+    },
```

Comment:
What goes in these? Looking at the [analytics schema](https://github.com/coveo/analytics_schema/tree/master/schemas/client), I don't see a "commerce search" event.

Reply:
I'm not sure: is there a plan to define one, or are we using the [regular search schemas](https://github.com/coveo/analytics_schema/tree/master/schemas/internal/api.search.search/2)?

@pvanool , thoughts?

Reply:
It's currently on pause. We definitely want to go forward with it in the future. Is this something that we can easily add when we have a schema ready?

Reply:
As stated in the PR description, we'll tackle this in CAPI-325 and CAPI-326